mobile/lib/shared/views/backup_controller_page.dart

```diff
@@ -89,13 +120,36 @@ class BackupControllerPage extends HookConsumerWidget {
                   : Container(),
               Padding(
                 padding: const EdgeInsets.only(top: 8.0),
-                child: OutlinedButton(
-                  onPressed: () {
-                    isAutoBackup
-                        ? ref.watch(authenticationProvider.notifier).setAutoBackup(false)
-                        : ref.watch(authenticationProvider.notifier).setAutoBackup(true);
-                  },
-                  child: Text("Turn $backupBtnText Backup", style: const TextStyle(fontWeight: FontWeight.bold)),
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: [
+                    DropdownButton<String>(
+                      hint: const Text("Select album to backup"),
+                      icon: const Icon(Icons.photo),
+                      isExpanded: true,
+                      items: assetEntities.hasData ?
+                      ((assetEntities.data as List<AssetPathEntity>)
+                          .map((AssetPathEntity element) => {
+                            DropdownMenuItem(
+                              value: element.id,
+                              child: Text(element.name),
+                            )
+                          }.first).toList())
+                          : null,
+                      value: assetEntities.hasData ? backupAlbumId : null,
```

Comment:
This line is causing issues on iOS when the app is first opened with this feature. I am not sure why yet

Reply:
Maybe it's the null `value` or the empty `items`, strangely it does work perfectly on Android. Will check with another device

Reply:
Ah, I actually figure it out. 

Refer to comment
[1](https://github.com/alextran1502/immich/pull/136/files#r862404345)
[2](https://github.com/alextran1502/immich/pull/136/files#r862404439)